docs/source/tutorials/multiple_pilots.ipynb

```diff
@@ -1,20 +1,420 @@
 {
```

Comment:
<p>I would reduce the number of the cores since this example runs on local host which could be someone's laptop</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
it is not physical resources, thus any number can be set. I've also added a comment regarding this